packages/ai-chat/src/chat/shared/containers/MessagesComponent.scss

```diff
@@ -56,3 +57,21 @@
   block-size: 100%;
   inline-size: 100%;
 }
+
+.WAC__messages--unreadMessagesIndicator {
```

Comment:
We actually have an "unread messages indicator" on the launcher to show you have received messages while you have it minimized. Just to avoid confusion can you make up a different name here?